docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+processor:
+  build: ../data-processor/.
+  links:
+    - queue
+    - db
+  environment:
+    VIP_DP_AWS_ACCESS_KEY:
+    VIP_DP_AWS_SECRET_KEY:
+    VIP_DP_S3_BUCKET:
+    VIP_DP_SQS_REGION:
+    VIP_DP_SQS_QUEUE:
+    VIP_DP_SQS_FAIL_QUEUE:
+    VIP_DP_RABBITMQ_EXCHANGE:
+queue:
```

Comment:
Let's call this `rabbitmq:` and rename the env vars in the `config.edn` appropriately to be consistent with other projects.